test/integration/events.load.spec.js

```diff
@@ -56,5 +56,31 @@ describe('integration:events:load', () => {
     expect(res.body).to.have.all.keys(['id', 'name', 'sessions']);
     expect(res.body.sessions[0]).to.have.all.keys(['id', 'eventId', 'name', 'description', 'status', 'tickets']);
   });
+  describe('when ICS', () => {
+    it('should return an ics when specifying the format by header', async () => {
+      const res = await request(app)
+        .get('/events/a60dc59d-2db2-4d5d-a6d3-c08473dee5d4')
+        .set('Accept', 'text/calendar')
+        .expect('Content-Type', /text\/calendar/)
+        .expect(200);
+      const attributes = res.text.split('\r\n');
+      expect(attributes[0]).to.equal('BEGIN:VCALENDAR');
+      expect(attributes[1]).to.equal('VERSION:2.0');
+      expect(attributes[2]).to.equal('CALSCALE:GREGORIAN');
+      expect(attributes[3]).to.equal('PRODID:adamgibbons/ics');
+      expect(attributes[4]).to.equal('METHOD:PUBLISH');
+      expect(attributes[5]).to.equal('X-PUBLISHED-TTL:PT1H');
+      expect(attributes[6]).to.equal('BEGIN:VEVENT');
+      expect(attributes[7]).to.equal('UID:a60dc59d-2db2-4d5d-a6d3-c08473dee5d4');
+      expect(attributes[8]).to.equal('SUMMARY:Test event 1');
+      expect(attributes[9]).to.match(/DTSTAMP:[0-9]+T[0-9]+Z/);
+      expect(attributes[10]).to.match(/DTSTART:[0-9]+T[0-9]+Z/);
+      expect(attributes[11]).to.match(/DTEND:[0-9]+T[0-9]+Z/);
+      expect(attributes[12]).to.equal('URL:https://zen.coderdojo.com/events/a60dc59d-2db2-4d5d-a6d3-c08473dee5d4');
```

Comment:
nice! This is a feature I like to have.

Reply:
Actually, that's wrong.
I have non-pushed changes, but the RFC is designed to take the URL as an updatable link to the ICS of the event, not an event page. 
I'll have to add the actual link to the description of the event as a .. workaround.